runners/flink/src/main/java/org/apache/beam/runners/flink/LyftFlinkStreamingPortableTranslations.java

```diff
@@ -70,6 +73,7 @@
       LoggerFactory.getLogger(LyftFlinkStreamingPortableTranslations.class.getName());
 
   private static final String FLINK_KAFKA_URN = "lyft:flinkKafkaInput";
+  private static final String FLINK_KAFKA_SINK_URN = "lyft:flinkKafkaSink";
```

Comment:
`lyft:flinkKafkaSink` gets a little bit confusing. Can it be `lyft:flinkKafkaProducer`?